telegram/bot.py

```diff
@@ -154,6 +154,8 @@ def decorator(self, *args, **kwargs):
             if result is True:
                 return result
 
+            self.create_references(result)
+
```

Comment:
why is it needed in the `message` decorator? isn't it only for sending messages? (after you already have a bot instance...)